packages/cli/lib/prompts/projectDevTargetAccountPrompt.js

```diff
@@ -12,133 +12,145 @@ const { logger } = require('@hubspot/local-dev-lib/logger');
-    let devTestAccountUsage = undefined;
-    try {
-      devTestAccountUsage = await fetchDeveloperTestAccounts(defaultAccountId);
-    } catch (err) {
-      logger.debug(
-        'Unable to fetch developer test account usage limits: ',
-        err
-      );
-    }
+  const sandboxAccounts = accounts
+    .reverse()
+    .filter(
+      config => isSandbox(config) && config.parentAccountId === defaultAccountId
+    );
+  let disabledMessage = false;
 
-    const devTestAccounts = accounts
-      .reverse()
-      .filter(
-        config =>
-          isDeveloperTestAccount(config) &&
-          config.parentAccountId === defaultAccountId
-      );
-    let disabledMessage = false;
-    if (
-      devTestAccountUsage &&
-      devTestAccountUsage.results.length >= devTestAccountUsage.maxTestPortals
-    ) {
-      disabledMessage = i18n(`${i18nKey}.developerTestAccountLimit`, {
+  if (sandboxUsage['DEVELOPER'] && sandboxUsage['DEVELOPER'].available === 0) {
+    if (sandboxAccounts.length < sandboxUsage['DEVELOPER'].limit) {
+      disabledMessage = i18n(`${i18nKey}.sandboxLimitWithSuggestion`, {
         authCommand: uiCommandReference('hs auth'),
-        limit: devTestAccountUsage.maxTestPortals,
+        limit: sandboxUsage['DEVELOPER'].limit,
+      });
+    } else {
+      disabledMessage = i18n(`${i18nKey}.sandboxLimit`, {
+        limit: sandboxUsage['DEVELOPER'].limit,
       });
     }
-
-    choices = [
-      ...devTestAccounts.map(mapNestedAccount),
-      {
-        name: i18n(`${i18nKey}.createNewDeveloperTestAccountOption`),
-        value: {
-          targetAccountId: null,
-          createNewDeveloperTestAccount: true,
-        },
-        disabled: disabledMessage,
+  }
+  // Order choices by Developer Sandbox -> Standard Sandbox
+  choices = [
+    ...sandboxAccounts
+      .filter(a => a.accountType === HUBSPOT_ACCOUNT_TYPES.DEVELOPMENT_SANDBOX)
+      .map(mapNestedAccount),
+    ...sandboxAccounts
+      .filter(a => a.accountType === HUBSPOT_ACCOUNT_TYPES.STANDARD_SANDBOX)
+      .map(mapNestedAccount),
+    {
+      name: i18n(`${i18nKey}.createNewSandboxOption`),
+      value: {
+        targetAccountId: null,
+        createNestedAccount: true,
       },
-    ];
-  } else {
-    let sandboxUsage = {};
-    try {
-      sandboxUsage = await getSandboxUsageLimits(defaultAccountId);
-    } catch (err) {
-      logger.debug('Unable to fetch sandbox usage limits: ', err);
-    }
+      disabled: disabledMessage,
+    },
+    {
+      name: i18n(`${i18nKey}.chooseDefaultAccountOption`),
+      value: {
+        targetAccountId: defaultAccountId,
+        createNestedAccount: false,
+      },
+    },
+  ];
 
-    const sandboxAccounts = accounts
-      .reverse()
-      .filter(
-        config =>
-          isSandbox(config) && config.parentAccountId === defaultAccountId
-      );
-    let disabledMessage = false;
+  return selectTargetAccountPrompt(
+    defaultAccountId,
+    'sandbox account',
```

Comment:
Does `sandbox account` not exist in `HUBSPOT_ACCOUNT_TYPE_STRINGS `?

Reply:
Yeah it unfortunately doesn't, the usage here needs to be the generic sandbox type but `HUBSPOT_ACCOUNT_TYPE_STRINGS` only has standard/development sandbox. IIRC this is the only spot I used this at and didnt know if it'd be worth creating a const for